fossunited/fossunited/doctype/foss_united_team/foss_united_team.py

```diff
@@ -3,6 +3,8 @@
 
 from frappe.model.document import Document
 
+from fossunited.doctype_ids import FOSS_USER_PROFILE
+
```

Comment:
We need to revert changes to this file. The changes are unrelated and don't need to be included in this PR.


```suggestion
```